### PR TITLE
Update uuid to 0.6

### PR DESCRIPTION
**Pull request by Dylan-DPC**
Hey

Updating this crate to 0.6 since the new release is out. There are no further changes required.

### CODE REVIEW

**Comment by matthiasbeyer** [comment]
Hey, thanks for you PR, this is awesome!

Unfortunately this breaks the build as `task-hookrs` uses uuid `0.5`. I will update `task-hookrs` [with this PR](https://github.com/matthiasbeyer/task-hookrs/pull/55) and after that we can push a commit here which updates the `task-hookrs` dependency.

**Comment by matthiasbeyer** [comment]
Hm, I don't want to release task-hookrs 0.5.0 just because of a dependency update. But I guess its okay to depend on task-hookrs master branch, at least until we get the 0.7.0 release of imag ready, so if you could push a commit which changes that dependency, I guess that would fix this PR. :+1:

**Comment by Dylan-DPC** [comment]
Alternatively you could release a 4.1 since I don't see any breaking change.

**Comment by matthiasbeyer** [comment]
A dependency update were types from the dependency are exported is a breaking change.

**Comment by Dylan-DPC** [comment]
Agreed. but if you are using your dependency and the way you use it doesn't change then it can't be considered breaking. However it is difficult to consider whether it is breaking or not its fine I will hook it to master but hopefully no change in master breaks the crate

File: lib/domain/libimagtodo/Cargo.toml [hunk 1]
```diff
@@ -20,7 +20,7 @@ is-it-maintained-open-issues      = { repository = "matthiasbeyer/imag" }
 maintenance                       = { status     = "actively-developed" }
 
 [dependencies]
-task-hookrs = "0.4"
+task-hookrs = { git="https://github.com/matthiasbeyer/task-hookrs.git" branch="master" }
```
Comment (matthiasbeyer) [inline lib/domain/libimagtodo/Cargo.toml]: There's a `,` missing before the `branch` key.

**Comment by matthiasbeyer** [comment]
Awesome! Feel free to squash all your changes into one commit! If you do this, please provide a commit message that explains your changes! <3

**Comment by Dylan-DPC** [comment]
@matthiasbeyer all good :+1:

**Pull request merged**